src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -54,7 +55,7 @@ public List<MovieSessionResponseDto> findAvailableSessions(@RequestParam Long mo
         return movieSessionService.findAvailableSessions(movieId, date)
                 .stream()
                 .map(movieSessionResponseDtoMapper::mapToDto)
-                .toList();
+                .collect(Collectors.toList());
```

Comment:
You may use Java 17 in this HW

Reply:
I tried and what I received: "ASM ClassReader failed to parse class file - probably due to a new Java class file version that isn't supported yet: file AppConfig.class. Unsupported class file major version 61"
That's why I changed to 11v. in pom.xml.
Can I fix this in another way?